routes/users.go

```diff
@@ -203,3 +218,83 @@ func GetUserHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	json.NewEncoder(w).Encode(user)
 }
+
+func PutUpdateUserHandler(w http.ResponseWriter, r *http.Request) {
+
+	// Checks for food truck ID
+	params := mux.Vars(r)
+	userID, userIDExists := params["userID"]
```

Comment:
Shouldn't a user only be able to update itself? We shouldn't need to specify the ID since we can just use the ID of the user that authenticated

Reply:
so no userID?

Reply:
oh i see what ur saying

Reply:
so remove this